### PR TITLE
MINOR: Do not end Javadoc comments with `**/`

### PR DESCRIPTION
**Pull request by mjsax**
Follow up to https://github.com/apache/kafka/pull/7034 that got abandoned.

### CODE REVIEW

**Comment by cadonna** [comment]
No javadocs are generated for non-public API and test classes. Do you still want to refactor them?

**Comment by mjsax** [comment]
I would say yes -- does not hurt, and avoids IntelliJ warnings. I like to get a green checkmark on files :)

**Review by bbejeck — APPROVED** [review]
LGTM

**Review by lihaosky — APPROVED** [review]
Thanks for the fix!

File: connect/api/src/main/java/org/apache/kafka/connect/health/ConnectClusterDetails.java [hunk 1]
```diff
@@ -27,6 +27,6 @@ public interface ConnectClusterDetails {
      * Get the cluster ID of the Kafka cluster backing this Connect cluster.
      * 
      * @return the cluster ID of the Kafka cluster backing this Connect cluster
-     **/
+     */
     String kafkaClusterId();
 }
```
Comment (jlprat) [inline connect/api/src/main/java/org/apache/kafka/connect/health/ConnectClusterDetails.java:32]: Not really relevant, but this file seems to miss a break line at the end of the file.

File: tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java [hunk 2]
```diff
@@ -290,7 +290,7 @@ record = new ProducerRecord<>(topic, key, value);
         }
     }
 
-    /** Returns a string to publish: ether 'valuePrefix'.'val' or 'val' **/
+    /* Returns a string to publish: ether 'valuePrefix'.'val' or 'val' */
```
Comment (jlprat) [inline tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java]: This is javadoc, right? I mean it's really vague but it documents the method below, so I think it misses a *
```suggestion
    /** Returns a string to publish: ether 'valuePrefix'.'val' or 'val' */
```

**Review by jlprat — COMMENTED** [review]
Thanks @mjsax 
I think one javadoc comment got converted to a comment by mistake.

File: connect/api/src/main/java/org/apache/kafka/connect/health/ConnectClusterDetails.java (hunk 1 above)
Comment (mjsax) [reply connect/api/src/main/java/org/apache/kafka/connect/health/ConnectClusterDetails.java:32]: It think it's just an encoding question between Linux / Window file system -- I guess it was automatically updated when I edited the file.

File: tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java (hunk 2 above)
Comment (mjsax) [reply tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java]: Not sure -- `VerifiableProducer` does not seem to be a class that users would instantiate? But happy to keep it as JavaDoc (the single line formatting indicated it's not a JavaDoc 🤷 )

**Pull request merged**